Makefile

```diff
@@ -27,4 +26,11 @@ test:
 	@docker run datamegh:test
 	@npm run typecheck
 
+format:
+	@black .
+
+check:
+	@black --check --diff .
+	@pyright
```

Comment:
I think this would not be needed now because `make test` does everything. We should just do `make test` before pushing our changes - it validates everything we need.

Reply:
I understand that make test would perform the above task. However, just to check if my code is formatted and typechecked properly developer would need to run the test multiple time. So make check would simply help the developer to check the changed code. In future, the test cases would increase so does the time to execute make test.

Reply:
I like `check`.

Reply:
@Basanta78 That make sense. How about having it in a watch mode instead then - because this is what you need while you're working with code. 

Is this possible?
```
$ black --check --diff --watch
```

Reply:
If it's not possible - then we can it's OK.

Reply:
I think black doesn't support watch mode. If we need such feature, we might integrate separate tools or write a separate script in the coming days.